File: LE6/d7-1.java
import java.util.Scanner;

public class Ex7_V1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double nota;
        System.out.print("Digite a nota (0 a 100): ");
        nota = sc.nextDouble();

        if (nota >= 90) {
            System.out.println("Classificação: A");
        } else if (nota >= 80) {
            System.out.println("Classificação: B");
        } else if (nota >= 70) {
            System.out.println("Classificação: C");
        } else if (nota >= 60) {
            System.out.println("Classificação: D");
        } else {
            System.out.println("Classificação: E");
        }
    }
}

File: LE6/d7-1.cpp
#include <iostream>
using namespace std;

int main() {
    double nota;

    cout << "Digite a nota (0 a 100): ";
    cin >> nota;

    if (nota >= 90) {
        cout << "Classificação: A" << endl;
    } else if (nota >= 80) {
        cout << "Classificação: B" << endl;
    } else if (nota >= 70) {
        cout << "Classificação: C" << endl;
    } else if (nota >= 60) {
        cout << "Classificação: D" << endl;
    } else {
        cout << "Classificação: E" << endl;
    }

    return 0;
}